docs/install-upgrade.md

```diff
@@ -170,6 +170,15 @@ for MongoDB Docker images:
 
 1. In the Operator [resource definition](../config/manager/manager.yaml), set the `MONGODB_IMAGE` and `MONGODB_REPO_URL` environment variables:
 
+   **NOTE:** Use the official 
+   [MongoDB Community Server images](https://hub.docker.com/r/mongodb/mongodb-community-server). 
```

Comment:
@slaskawi @nammn is this correct? I thought we have official images in quay only?